editor/plugins/script_text_editor.cpp

```diff
@@ -2462,6 +2470,7 @@ void ScriptTextEditor::register_editor() {
 	ED_SHORTCUT("script_text_editor/evaluate_selection", TTR("Evaluate Selection"), KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::SHIFT | Key::E);
 	ED_SHORTCUT("script_text_editor/toggle_word_wrap", TTR("Toggle Word Wrap"), KeyModifierMask::ALT | Key::Z);
 	ED_SHORTCUT("script_text_editor/trim_trailing_whitespace", TTR("Trim Trailing Whitespace"), KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::ALT | Key::T);
+	ED_SHORTCUT("script_text_editor/trim_final_newlines", TTR("Trim Final Newlines"), KeyModifierMask::CMD_OR_CTRL | KeyModifierMask::SHIFT | Key::T);
```

Comment:
I've added this to have parity with `Trim Tailing Whitespace`, but I'm not sure that it has the same utility. I could see removing this, or possibly defaulting to no keybind.

Reply:
While waiting for the cursor PR to be merged, @AThousandShips, do you have any preference on this addition?

Reply:
No opinions here on my part

Reply:
Cool, I think that I will add it as a potential keybind with no default since as @Calinou suggested this feature will be on by default.